content/sensu-go/6.2/sensuctl/_index.md

```diff
@@ -41,6 +58,22 @@ The default URL is `http://127.0.0.1:8080`.
 To connect to a [Sensu cluster][4], connect sensuctl to any single backend in the cluster.
 For information about configuring the Sensu backend URL, see the [backend reference][3].
 
+### sensuctl configure flags
+
+Run `sensuctl configure -h` to view command-specific flags you can use to set up sensuctl and bypass interactive mode.
+The following table lists the command-specific flags.
+
+| Flag | Function and important notes
+| ---- | ----------------------------
+`--format` | Preferred output format (default "tabular"). String.
+`-h` or `--help` | Help for the configure command.
+`-n` or `--non-interactive` | Do not administer interactive questionnaire.
+`--oidc` | Use an OIDC provider for authentication (instead of username and password).
+`--password string` | User password. String.
+`--port` | Port for local HTTP webserver used for OAuth 2 callback during OIDC authentication (default 8000). Integer.
+`--url` |  The Sensu backend url (default "http://127.0.0.1:8080"). String.
+`--username` | Username. String.
+
```

Comment:
If you bypass interactive mode, how does sensuctl know which namespace you want to use?

Reply:
You either specify the `--namespace` global flag, or it will fallback to the default namespace, which is `default`.

Reply:
Got it, thank you